examples/js/tests/authorization.ts

```diff
@@ -417,10 +491,14 @@ function restAssertAuthenticationNotSatisfactory(error) {
 }
 
 async function main() {
+  // disable logging
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  console.log = () => {};
```

Comment:
That one seems a bit scary :grimacing:
We can leave it like that for now, but I would suggest having a central logger which can filter out logs based on a LOG_LEVEL env

Reply:
Agreed! Let's leave it like this for now though, since we're still in the prototyping phase. I've created [an issue](https://github.com/gcomte/dingir-exchange/issues/27) for us to keep track of it.